airbyte-integrations/connectors/destination-iceberg-v2/src/main/kotlin/io/airbyte/integrations/destination/iceberg/v2/io/RecordWrapper.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.iceberg.v2.io
+
+import org.apache.iceberg.data.Record
+import org.apache.iceberg.types.Types
+
+/**
+ * Wraps the Iceberg [Record] class to add the potential delta [Operation] associated with the
+ * record. All record operations are delegated to the provided [Record] object.
+ */
+data class RecordWrapper(val delegate: Record, val operation: Operation) : Record {
```

Comment:
I think this entire class can maybe just be `data class RecordWrapper(val delegate: Record, val operation: Operation) : Record by delegate`

Reply:
Done